pkg/datastoresrp/setup/operations.go

```diff
@@ -0,0 +1,162 @@
+/*
+Copyright 2023 The Radius Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package setup
+
+import v1 "github.com/radius-project/radius/pkg/armrpc/api/v1"
+
+var operationList = []v1.Operation{
+	{
+		Name: "Applications.Datastores/operations/read",
+		Display: &v1.OperationDisplayProperties{
+			Provider:    "Applications.Datastores",
+			Resource:    "operations",
+			Operation:   "Get operations",
+			Description: "Get the list of operations",
+		},
+		IsDataAction: false,
+	},
+	{
+		Name: "Applications.Datastores/redisCaches/read",
+		Display: &v1.OperationDisplayProperties{
+			Provider:    "Applications.Datastores",
+			Resource:    "redisCaches",
+			Operation:   "List redisCaches",
```

Comment:
Should this be like below?
```suggestion
			Operation:   "List Redis Caches",
```

Reply:
This question applies to other resources too.

Reply:
Giving more context. The changes are based upon updates in CoreRP and the text in descriptions is similar to this file eg: https://github.com/radius-project/radius/blob/9f277180f3ba3da7fda5169b3303fab7f3f26538/pkg/corerp/setup/operations.go#L107 
I would request to continue with the format as in CoreRP or discuss offline and we make updates across the board for all RPs.

Reply:
I think we should file an issue to get back to this. Because `List httproutes` also doesn't look right since it doesn't follow the camel case pattern.